vagrant/provisioning/roles/common/tasks/main.yml

```diff
@@ -29,6 +36,18 @@
       - python3
       - python3-pip
 
+- name: copy requirements.txt needed for pip2 upgrade
+  become: yes
+  copy:
+    src: requirements.txt
+    dest: /home/
```

Comment:
`/home` is not a good directory for such a file. This is clearly a temporary file, so it would be much better to put it in `/tmp`.

Reply:
You're right @fabricetriboix , I hesitated between these 2 directories :)